Example/InstanceID/Tests/FIRInstanceIDTokenInfoTest.m

```diff
@@ -69,10 +69,15 @@ - (void)tearDown {
 
 - (void)testTokenInfoCreationWithInvalidArchive {
   NSData *badData = [@"badData" dataUsingEncoding:NSUTF8StringEncoding];
+  FIRInstanceIDTokenInfo *info = nil;
+  @try {
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wdeprecated-declarations"
-  FIRInstanceIDTokenInfo *info = [NSKeyedUnarchiver unarchiveObjectWithData:badData];
+    info = [NSKeyedUnarchiver unarchiveObjectWithData:badData];
```

Comment:
I think `XCTAssertThrows` could work well here too, but this works as well for the double validation :)